QEfficient/transformers/modeling_utils.py

```diff
@@ -117,6 +117,7 @@
         GPT2LMHeadModel.__name__,
         GPTJForCausalLM.__name__,
         MptForCausalLM.__name__,
+        "MPTForCausalLM",
```

Comment:
instead of this can we do, if full_batch_size and qeff_model.model.config.architectures[0].lower() not in [s.lower() for s in get_lists_of_cb_qeff_models.architectures]: in line 323 of QEfficient/exporter/export_hf_to_ai_100.py